src/pages/docs/concepts/ucan.md

```diff
@@ -0,0 +1,294 @@
+
+First, make a text file called `hello.txt` with some simple content:
+
+```shell
+echo "Hello, web3!" > hello.txt
+```
+
+Now you can use `w3 up hello.txt` to upload the file. You should see something like this:
+
+```
+  1 file (12B)
+  bagbaieraspawtgooy5lptr7loyd3fxjsrgkamre3y6au3ga4df5bkhrxdkmq
+⁂ Stored 1 file
+⁂ https://w3s.link/ipfs/bafybeidwm2gfdrx2yjnzdqnw7n3a2er2vo5uwv6w5otjfbyboeoq3jfiby
+
+```
+
+Running `w3up list` should now show the CID of your upload:
+
+```
+bafybeidwm2gfdrx2yjnzdqnw7n3a2er2vo5uwv6w5otjfbyboeoq3jfiby
+```
+
+You can also use the HTTP gateway at `w3s.link` to quickly fetch the content and check that it resolves. With the CID above, the gateway URL would be `https://w3s.link/ipfs/bafybeidwm2gfdrx2yjnzdqnw7n3a2er2vo5uwv6w5otjfbyboeoq3jfiby`.
+
+### 4. Get the DID of the agent you want to delegate to
+
+Switch over to your second terminal window with the `W3_STORE_NAME` variable set and run the `w3 whoami` command. You should see a string starting with `did:key:`. Copy that string to your clipboard for the next step.
+
+##### 5. Delegate access to the new Agent DID
+
+Run the `w3 delegation create` command, passing in the DID for the second agent. The `-o` flag tells the `w3` command to write the delegation to a file named `delegation.car`:
+
+```shell
+w3 delegation create -o delegation.car SECOND_AGENT_DID
+```
+
+Be sure to replace `SECOND_AGENT_DID` in the example command with the DID for the second agent that you copied in step 4.
+
+This will create a file called `delegation.car` in your current directory. 
+
+##### 6. Add the delegated space to the second agent
+
+Next, import this delegation to our second agent.
+
+In the second terminal window, with the `W3_STORE_NAME` variable set, use the `w3 space add` command, passing in `delegation.car`:
+
+```shell
+w3 space add ./delegation.car
+```
+
+If you now run `w3 space ls` in the second terminal, you should see the space you just imported in the list.
+
+Imported spaces are not automatically set as the default space, so we need one more command before we're ready to go. Copy the space DID from the `w3 space ls` command you just ran and run this to set the imported space as the default for uploads:
+
+```bash
+w3 space use SPACE_DID
+```
+
+Replace `SPACE_DID` with the space DID you just copied. If you run `w3 space ls` again, you should see a `*` symbol next to the space, indicating that it's currently active.
+
+##### 7. Upload to the space from the second agent
+
+Now that your second agent has access to the space created by the first, you can use `w3 up` in your second terminal window to upload whatever you like.
+
+Note that the `w3 delegation create` command we used in step 5 delegates the `'*'` or "top" capability by default, so the second agent will have full access to the space. You can restrict the set of delegated capabilites using the `--can` flag. Try running `w3 delegation create --help` to see more options.
+
+## Wrapping up
```

Comment:
we have a few other resources that we could reference here that cover how UCANs work and practical system implementations:
- ucantoo workshop talk/video in Lisbon
- Where is my CAR talk in Fil dev summit